cirq-core/cirq/testing/consistent_controlled_gate_op_test.py

```diff
@@ -76,3 +76,14 @@ def test_assert_controlled_and_controlled_by_identical():
         cirq.testing.assert_controlled_and_controlled_by_identical(
             GoodGate(), num_controls=[1, 2], control_values=[(1,), (1, 1, 1)]
         )
+
+
+def test_assert_controlled_unitary_consistent():
+    cirq.testing.assert_controlled_and_controlled_by_identical(
+        GoodGate(exponent=0.5, global_shift=1 / 3)
```

Comment:
Worth testing with global_shift=0, or is that covered somewhere else?

Reply:
`global_shift = 0` is a strictly simpler case. And this is a test to make sure `assert_controlled_and_controlled_by_identical` is doing the right thing, not to test the correctness of `GoodGate` and `BadGate`. We have tests for `global_shift=0` and `global_shift != 0` for `CCX` / `CCZ` etc., so I think this should be sufficient.